### DISCUSSION TITLE
Set the name of `default` organization?

### DISCUSSION THREAD

**Discussion by fdcastel** [openobserve/openobserve#4893] [Q&A] [answered]
There is a way to rename the `default` organization?

Or remove it?

Or (better yet): to add an [ENV VAR](https://openobserve.ai/docs/environment-variables/#common) to preset the first organization name instead of `default`?

**Comment by fdcastel** [comment]
Also, the `ZO_GRPC_ORG_HEADER_KEY` [environment variable](https://openobserve.ai/docs/environment-variables/#common) appears to have the default value of `organization` (and not `openobserve-org-id` as that page says).

I don't have this environment variable set in my test environment. But, when I go to `/web/ingestion/recommended/traces` of my OpenObserve instance it shows:

![image](https://github.com/user-attachments/assets/9f92dbe8-7888-4346-b8e3-66968d6122e1)

**Comment by Manazsharma** [comment, marked as answer]
Currently, there is no way to remove/rename an org. Users can create a new org by modifying the curl command <org>/<stream>.

**Reply by Orholt** [reply]
Any further explanation behind this?
Is it "by design" or "not implemented" ?

**Reply by Manazsharma** [reply]
This is on the roadmap, but no ETA atm.
